server/server.go

```diff
@@ -1474,3 +1474,43 @@ func (server *HTTPServer) getClusterListAndUserData(
 
 	return
 }
+
+// getWorkloadsForOrganization returns a list of workloads for given organization ID.
```

Comment:
I'd put an explanation when `nil` and when empty slice is returned

Reply:
done, good idea